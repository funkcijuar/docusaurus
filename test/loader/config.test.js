const path = require('path');
const fs = require('fs');
const loadConfig = require('../../lib/loader/config');

describe('loadConfig', () => {
  const simpleDir = path.join(__dirname, '__fixtures__', 'simple');
  const customDir = path.join(__dirname, '__fixtures__', 'custom');

  test('simple', () => {
    expect(loadConfig(simpleDir)).toMatchInlineSnapshot(`
Object {
  "description": "Hello World",
  "title": "Hello World",
}
`);
  });

  test('custom', () => {
    expect(loadConfig(customDir)).toMatchInlineSnapshot(`
Object {
  "base": "blogi",
  "description": "Hello World",
  "dest": "blogi",
  "title": "Hello World",
}
`);
  });
});
