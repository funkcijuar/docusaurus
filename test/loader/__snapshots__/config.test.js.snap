// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadConfig custom 1`] = `
Object {
  "base": "blogi",
  "description": "Hello World",
  "dest": "blogi",
  "title": "Hello World",
}
`;

exports[`loadConfig simple 1`] = `
Object {
  "description": "Hello World",
  "title": "Hello World",
}
`;
